StreamVideoTests/WebRTC/SFU/SFUMiddleware_Tests.swift

```diff
@@ -8,89 +8,89 @@ import XCTest
 
 final class SFUMiddleware_Tests: XCTestCase, @unchecked Sendable {
 
-    private var mockSFUService: MockSignalServer!
```

Comment:
shouldn't we just delete this file?

Reply:
We will end up removing it, but before doing i would like to write the tests for the new SFUEventAdapter and ensure that all the cases in this file, will be covered in the new tests.